# for collision sensors read > 0 = collison
# for distance sensors read = distance
# for pressure sensors read = pressure
# etc.
float32 read
int16 sensorID